copier.yml

```diff
@@ -55,69 +68,90 @@ open_source_license:
 copyright_year:
   type: str
   help: "Copyright year(s)"
-  default: "2022"
+  default: "{% now 'utc', '%Y' %}"
 
 documentation:
   type: bool
   help: "Generate documentation?"
   default: yes
 
-has_cli:
+setup_venv:
   type: bool
-  help: "Does the project have a CLI?"
-  default: no
+  help: "Setup a virtual environment?"
+  default: yes
 
-cli_name:
+venv_version:
   type: str
-  help: "The name of the CLI"
-  default: "{{ project_slug }}"
-  when: "{{ has_cli }}"
+  help: "Python version for the virtual environment"
+  default: "3.11"
+  when: "{{ setup_venv }}"
 
 run_poetry_install:
   type: bool
   help: "Run poetry install after {{ package_name }} generation?"
-  default: no
+  default: yes
+  when: "{{ setup_venv }}"
 
 initial_commit:
   type: bool
   help: "Create an initial commit with the generated {{ package_name }}?"
-  default: no
+  default: yes
 
 setup_github:
   type: bool
   help: "Setup GitHub repository (requires gh CLI)?"
-  default: no
+  default: yes
+  when: "{{ initial_commit }}"
 
 setup_pre_commit:
   type: bool
   help: "Setup pre-commit hooks (requires pre-commit)?"
-  default: no
+  default: yes
+  when: "{{ setup_venv }}"
 
 add_me_as_contributor:
   type: bool
   help: "Add me as a contributor?"
+  default: yes
+  when: "{{ initial_commit }}"
+
+open_with_vscode:
+  type: bool
+  help: "Open with VSCode?"
   default: no
```

Comment:
Sorry, I don't use VSCode, so no. I'm not interested in making this question specific to a given IDE/editor either, but would be interesting if there is convention in env variable to open in the user's preferred editor: https://unix.stackexchange.com/q/4859

Reply:
Changed to use $VISUAL